src/msw/webview_ie.cpp

```diff
@@ -853,25 +857,229 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
-void wxWebViewIE::RunScript(const wxString& javascript)
+wxWebViewIEEmulationLevel wxWebViewIE::GetEmulationLevel()
+{
+    wxRegKey key(wxRegKey::HKCU, _T("SOFTWARE\\Microsoft\\Internet Explorer\\Main\\FeatureControl\\FEATURE_BROWSER_EMULATION"));
```

Comment:
Wouldn't it be better to create and use a constant for the registry path instead of using the same (long) string literal 6 times?